src/index.js

```diff
@@ -1,13 +1,45 @@
+/* eslint-disable */
 import _ from 'lodash';
+/* eslint-enable */
+
 import './style.css';
 
-function component() {
-  const element = document.createElement('div');
+const tasks = [
+  {
+    description: 'Reapir the car',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'Copmplete the Projects',
+    completed: false,
+    index: 2,
+  },
+  {
+    description: 'Submit the project',
+    completed: false,
+    index: 3,
+  },
+];
 
-  // Lodash, now imported by this script
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
+const displayTasks = () => {
+  let result = '';
+  tasks.filter((tasks) => tasks.index > 0).forEach((task) => {
+    result += `
+    <div class="item-container">
+      <div class="do-items">
+        <input type='checkbox' name='${task}'>
+        <label for=task>${task.description}</label>
+        <div class="item-dlt-btn">
+        <i class="fa-solid fa-xmark"></i>
+      </div>
+        <br>
+      </div>
 
-  return element;
-}
+    </div>
+    `;
+  });
+  document.getElementById('task').innerHTML = result;
+};
```

Comment:
- [ ] Kindly ensure your layout matches the provided one.

  Your Implementation | Required layout
  :---:|:---:
  ![eieo](https://user-images.githubusercontent.com/36542986/206353387-cfba0c44-f977-43fe-936a-e53d6214006f.jpg) | ![kdkd](https://user-images.githubusercontent.com/36542986/206353403-e6c67c49-5546-4e2a-8d68-b497fb9d0304.jpg)